src/Configuration/Route.php

```diff
@@ -39,6 +39,16 @@ public function getService()
         return $this->service;
     }
 
+    public function setLocalizedPaths(array $localizedPaths)
```

Comment:
This change does not seem to be related.

Reply:
Yes, not connected. But tests were falling.